app/assets/stylesheets/custom.css.scss

```diff
@@ -44,6 +52,45 @@ body {
   }
 }
 
+.smfont {
+  font-size: 0.9em;
+}
+
+.warning {
+  color: $red;
+  padding-left: 30px;
+}
+
+.card {
+  background-color: $white;
+  border: 1px solid transparent;
+  border-color: #BCE8F1;
+  border-radius: 4px;
+  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.05);
+}
+
+.card_edit {
+  border-color: $dark-blue;
+}
+
+.card__heading {
+  color:$white;

```

Comment:
Properties should be ordered background-color, border-bottom, border-color, border-top-left-radius, border-top-right-radius, color, padding<br>Colon after property should be followed by one space